### PR TITLE
Update benchmark comparison script and docs

### PR DESCRIPTION
**Pull request by harendra-kumar**
Also add a changelog in unicode-data README for reference.

### CODE REVIEW

File: benchmark/README.md [hunk 1]
```diff
@@ -1,21 +1,60 @@
...
+$ cabal run chart --flag bench-show
+```
+
+# Comparing with ICU (text-icu package)
+
+Install the icu library:
+
+```
+# On Mac OS using MacPorts
+$ sudo port install icu
```
Comment (Bodigrim) [inline benchmark/README.md:34]: Alternatively `brew install icu4c`

File: benchmark/README.md [hunk 2]
```diff
@@ -1,21 +1,60 @@
...
+
+# On Debian Linux distributions
+$ sudo apt-get install icu
+```
+
+If cabal cannot automatically find the icu library (e.g. when installed
+via macports install) then use explicit `LIBRARY_PATH` to tell it where
```
Comment (Bodigrim) [inline benchmark/README.md:41]: Another option is to pass it via command line flags:
```
cabal bench --extra-lib-dirs=/usr/local/opt/icu4c/lib --extra-include-dirs=/usr/local/opt/icu4c/include
```

File: benchmark/README.md (hunk 1 above)
Comment (harendra-kumar) [reply benchmark/README.md:34]: I use macports, I know that works. I can add the brew example as well if you know it works. BTW why is it `icu4c` with brew and not just `icu`?

File: benchmark/README.md (hunk 2 above)
Comment (harendra-kumar) [reply benchmark/README.md:41]: I used exactly those flags but they did not work for me. LIBRARY_PATH worked. Not sure why, but looks like LIBRARY_PATH is a safer option unless someone reports that it does not work for them.

File: benchmark/README.md (hunk 1 above)
Comment (Bodigrim) [reply benchmark/README.md:34]: I checked it yesterday, `brew install icu4s` works fine for me. 
(I do not know why it is not just `icu` %)

Comment (harendra-kumar) [reply benchmark/README.md:34]: Can you confirm if it is `icu4s` or `icu4c`? Earlier you wrote `icu4c` in all the command examples.

**Pull request merged**